### ISSUE TITLE
chunking without synthetic padding

### ISSUE THREAD

**Issue by jeff-regier** [prob-ml/bliss#275]
This code shows a way to divide an SDSS frame into "padded chunks". Each pixel belongs to at most one chunk, but a pixel may be contained in as many as four padded chunks. Pixels on the border of an SDSS frame are not part of any chunk. No synthetic padding is necessary, and the code should run efficiently.

```
ch = 200  # chunk height
cw = 300  # chunk width
cp = 24  # chunk padding

ihic = image.size(0) // ch  - 1  # image height in chunks
iwic = image.size(1) // cw  - 1  # image width in chunks

padded_chunks = torch.zeros((ihic, iwic, ch + 2 * cp, cw + 2 * cp))

for i in range(ihic):
    for j in range(iwic):
        h, w = i * ch + cp, j * cw + cp
        print(h, w)
        padded_chunks[i, j] = image[(h - cp):(h + ch + cp), (w - cp):(w + cw + cp)]

# TODO: 1. infer catalogs for all padded chunks
# TODO: 2. discard catalog entries that are outside the chunk (i.e., that are in the padding)
# TODO: 3. merge the catalogs
# TODO: 4. score the merged catalog (excluding any coadd objects that are not within a chunk)
```
